_studio/shared/src/cm_mem_copy.cpp

```diff
@@ -3221,6 +3225,11 @@ mfxStatus CmCopyWrapper::CopyVideoToSys(mfxFrameSurface1 *pDst, mfxFrameSurface1
 
         mfxI64 verticalPitch = (mfxI64)(pDst->Data.UV - pDst->Data.Y);
```

Comment:
looks like this code should look in following way:
```
mfxI64 verticalPitch = (mfxI64)(pDst->Data.UV - pDst->Data.Y);
if (pDst->Info.FourCC == MFX_FOURCC_RGBP)
{
    verticalPitch = (mfxI64)(pDst->Data.G - pDst->Data.B);
}
verticalPitch = (verticalPitch % dstPitch)? 0 : verticalPitch / dstPitch;
```

Reply:
+ this code should be covered with define as well:
```
mfxI64 verticalPitch = (mfxI64)(pDst->Data.UV - pDst->Data.Y);
#ifdef MFX_ENABLE_RGBP
if (pDst->Info.FourCC == MFX_FOURCC_RGBP)
{
    verticalPitch = (mfxI64)(pDst->Data.G - pDst->Data.B);
}
#endif
verticalPitch = (verticalPitch % dstPitch)? 0 : verticalPitch / dstPitch;
```

Reply:
done